lightning/src/chain/chainmonitor.rs

```diff
@@ -262,6 +345,54 @@ where C::Target: chain::Filter,
 		self.monitors.write().unwrap().remove(funding_txo).unwrap().monitor
 	}
 
+	/// Indicates the persistence of a [`ChannelMonitor`] has completed after
+	/// [`ChannelMonitorUpdateErr::TemporaryFailure`] was returned from an update operation.
+	///
+	/// Thus, the anticipated use is, at a high level:
+	///  1) This [`ChainMonitor`] calls [`Persist::update_persisted_channel`] which stores the
+	///     update to disk and begins updating any remote (e.g. watchtower/backup) copies,
+	///     returning [`ChannelMonitorUpdateErr::TemporaryFailure`],
+	///  2) once all remote copies are updated, you call this function with the update_id that
+	///     completed, and once all pending updates have completed the Channel will be re-enabled.
```

Comment:
We could put some `//` comments that point out that the Channel will be re-enabled as soon as all `MonitorUpdateId { contents: MonitorUpdate::MonitorUpdateId }` pending updates have completed (i.e. there can still be SyncPersist pending updates remaining)

Reply:
Ping?